### ISSUE TITLE
Submission: Echoes of the Red Dig Site

### ISSUE THREAD

**Issue by whitewolf87** [MoreEventsMod/More_Events_Mod_Beta#527]
**Echoes of the Red**

Note: Starts in a system with a Red Giant. If we can possibly kick up the possibility of failure to 11 that’d be awesome possum. 

Preliminary scans of [planet] indicate a canyon with several openings throughout. We originally thought they may be dwellings for some or another type of wildlife, but we now are not so sure as one of [scientist’s] interns realised that they are spaced almost equidistantly apart from one another. Not only that but each one—though poorly constructed—is a square too similar to the others to be considered a natural construction. 

Two probes sent down show a heavy door rolled over a second entrance about 100 meters into each cave, very well preserved. Unknown glyphs and runes line the walls leading to these entrances completely floor-to-ceiling.

GFX_evt_underground_civilization

[Send a ground team and start standard excavation procedures]

**Stage 1 (Difficulty=2)** GFX_archaeology_runes_D3

We began with four different archways located in the most accessible points within the canyon. The entrance is heavy, but not immobile. Pretty typical archaeological procedure was able to roll the first one aside. Though the others are proving a bit more stubborn, it’s believed that a bit more prodding will eventually yield the intended result. Disturbingly, we did find tell-tale triggers for several traps that are more likely than not still functional. We need to be extremely careful; some of these look quite nasty indeed. 

For now, we can focus our attention on the first room. It’s a tomb we’ve dated pretty far back to the end of this culture’s stone age. It looks to be completely untouched except for dust and contains a multitude of small personal items and clay jars filled with the remnants of food. First speculation of course is that these are the occupant’s intended supplies to help them into the afterlife. Translators working on the text on the rough stone sarcophagus in the center of the room have given us their initial work, and it reads: “Forever rests Xulathun X, Shaman of the Red and the highest of highs.” 

However, text in a different language was found on the archway above and behind the tomb in clear view of the door. This was remarkably easier to translate and just simply read: “You can go home”. One of the techs suggested earlier grave robbers trying to pull a joke, but if that’s the case, why didn’t they take anything? 

GFX_evt_discovered_hidden_door

[Maintain work on the other entrances]

Failure: While trying to work the stone, one of the student researchers fell into one of the oldest tricks in the book. They tripped a pressure plate and were immediately rewarded with an impalement that shocked the whole team to the core. They didn’t die right away, screaming and begging to do so until one of the escorts put them out of their misery without orders. The escort has expressed a desire to be shipped to [random colonized planet], which has been granted. Counseling will be provided. 

GFX_evt_collapsing_roof

[Contact the families. Provide restitution.] 

**Reward**: +1 minor relic. 

**Stage 2 (Difficulty=6)** GFX_archaeology_runes_C1

Almost immediately after we managed to gain entry to the first tomb, the second doorway started to give way under our techniques and then was easily rolled aside. However, this then gave way to another door down a short hallway. This next door is cast in mostly red marble with an inscription above in a language totally divorced from that of the first tomb, the script more rigid and angled. Even a cursory examination made it apparent that we’re looking at two completely different culture sets. 

Work began on opening this door and proceeding inside. The locks were robust, but not terribly difficult to figure out with some basic fiddling. Multiple team members reported hearing voices, though none of them appear to have heard the same set of phrases. None of these voices have been recorded on any piece of equipment, though multiple team members have tried. Multiple traps were identified, most of them triggered by tripwire. 

The electric lights that powered on upon entering the tomb were surprising as were the remarkably clean air readings. The sarcophagus itself in this case was encased in glass all the way around allowing a clear view of the mummified corpse within, a mammalian creature similar in height to us, with neatly-groomed facial hair and wearing a rotted and—for such a revered leader—simple garment. The translators have finally worked out the script that is over the entrance to this tomb, which reads simply “Lenin” and also in a plaque on the wall in here reading “Revolutioneer. Hero of the Supreme Soviet. Lover of the Red”. Above the tomb itself was text in the same style as the cryptic message in Xulathun X’s tomb reading simply “you can still go home”. 

GFX_evt_excavation_team

(Note: I'd love to actually use a picture of Lenin's Tomb for this one if it could be arranged. I'm willing to do it myself if you could give me a quick run-through on how to make an even picture)

**Failure 1**: Feeling around while examining the door, a researcher reported feeling a directional tuft of air, followed by the sounds of screaming and weapons fire from what sound files correspond to roughly industrial age weaponry. The entire group was found slaughtered upon further investigation with a single fully-loaded firearm placed in between their bodies as if to be some kind of statement. The ammunition canister was fully loaded and in a curved shape. No signs of the assailants were found. 

GFX_evt_interior_battle

[What could have possibly happened?!] (-5 clues, +1 relic)

**Failure 2**: The paleontology consultant at base camp, while examining what was thought to be a fossil, exploded into a fiery rage screeching about the rights of the workers and the upheaval of the monied classes. He then picked up a hatchet and proceeded to hack the camp provisions officer to death in front of his son, an intern student from the university on [random colony]. The officer’s son lunged at the consultant and received the hatchet buried in his forehead for his trouble. He was finally subdued by the guards who, negligent in their duties, were outside the camp having a break. The entire away team is troubled to say the least. The object the consultant was examining turned out to be a small fossilized piece of jewelry, most likely a lapel; a small, red shiny pentagram.

GFX_evt_zro_5

[Fire the guards. Inform the families] (-7 clues, +1 relic)

**Reward**: +3 minor relics, +200 society research

**Stage 3 (Difficulty=7)** GFX_archaeology_runes_C3

The team lead wrenched back as she managed to slide the stone away from the next door. She was greeted with a huge wall of water! Past that however we were clearly able to see inside to a room that seems to be somewhat larger than the last two. After a few hours of deliberation it was agreed that we should proceed forward with diving equipment, an unexpected but quickly fulfilled request. 
After donning our bathing costumes, we entered the tomb and everyone agrees this one easily wins the award for the most beautiful. A large stone sarcophagus lies against the northern wall, before which are two smaller stone sarcophagi and twelve still smaller ones in neat rows that alternate directions so if lines were drawn they are in a perpendicular grid. It would appear that each one was treated to grow algae and corral, and there are waterproofed sunlamps adoring the roof to help maintain the growth; the source of power for these has not yet been determined. We seeeautiful intricate pattern spawning twisted branches of colour and material winding around and under each other in a beguiling intertwine of stalks. Many colors are present but predictably the dominant one here is red and its various shades. Still immaculately beautiful. 

As with the previous tombs there is an inscription on the wall, these inlaid with crimson bio luminescent microorganisms seemingly to make the words glow. It reads: “Sacrificed here for the Red is the Lifetime Archduke of the Oceans Glublureum of Saltsand and his wives and children. Given unto the Red for peace and fulfillment, loved by his supplicants, defeater of the prawn beasts”. And on the opposite wall in the same script as the other warnings. “Next comes the bargain. You can still go home.”

At this point, a large sinkhole has opened up in the middle of the dig site taking a small bulldozer that was parked there near it. Looking down into it, we can see a circular stone floor with what is most likely a handle in the middle of it. Spelunking would easily drop us down onto the surface. 

GFX_evt_underwater_ruins

[Make it so]

**Failure**: After getting the door to the tomb open, one or our crew leaned up against a wall and managed to trigger some kind of switch which dumped onto everybody’s heads a multitude of relatively benign looking but actually very vicious crabs. The pincers were quite poisonous. After drawing back the team, we have a total death count of 4 with all 9 others injured. The bodies are severely bloated and hemorrhaging and it’s debatable how many survivors will make it through the night let alone the next week. 

GFX_evt_emergency_workers

[Appalling] (-5 clues)

**Reward**: +500 society research

**Stage 4 (Final, Difficulty=1)** GFX_archaeology_runes_C5

Above the large but simple latching mechanism on the door are inscribed the words: “you may still go home” in the script common to all the tombs. Once opened, the doors started to slide out, as we heard some shrill screaming followed by complete loss of contact with the ground team and the orbiting ship. Even more troubling, on [Home Planet], our esteemed leader [Leader Name] gripped his/her chest and keeled over screaming in pain before disappearing apparently into thin air. 

All members of the excavation are at this point presumed dead and all equipment is presumed destroyed. There was however an energy spike recorded coming from the canyons surrounding the sinkhole valley just before the sensor probes shut down. 

GFX_evt_warm_barren

**Failure**: None

**Reward**: None, [Scientist] dies, [Science Ship] is lost. Spawns special project at planet to determine what happened. [Civ Leader] dies and is replaced according to government type. 

**Special Project**

(Requires science ship in orbit with scientist skill level 3, takes 360 days)

The tragic and mysterious disappearance of the away team calls for a rescue and reconnoiter mission to determine anything we can. 

GFX_evt_landing_ship

[Proceed with due caution]

(Upon completion)

While nothing immediately threatening happened while at the dig site, what we found there is infinitely more disturbing. A new tomb had opened and unlike the others this time the door had been conveniently left open for us. Stepping within we observed our own architecture, and our own burial practices…containing the reclined body of [Civ Leader] just as we lost saw them. His palms are open, offering The Stone to us. Come to think of it, we aren’t disturbed at all. This was our Bargain. We will be happy with it. 

We are happy with it

We closed their coffin because it was right for the Red. 

We carried out the service right there and then for the Red. 

Our new sigil is beautiful. 

This stone will bring us joy and plenty. 

GFX_evt_mem_brainworm

**Rewards**: The banner and colour scheme of the civilization is changed to some variant of red. This effect is instantaneous. +20 spiritualist ethics attraction on all pops. +20 authoritarian ethics attraction all pops. 

**Relic Awarded**: The Stone: A seemingly inconspicuous hunk of iron laded basalt with a slight glow to it that seems more black than anything else. 

When activated, sacrifices the national leader, however grants +20 happiness to all pops for the duration of the effect and +5% menial worker output, and +2% specialist output.

**Comment by whitewolf87** [comment]
### **Alternate Text for Gestatls**

### **Note: WiP because I'm considering the Gestalts and machines having a completely alternate path past stage 3**

Excavation Site Idea: Echoes of the Red

Note: Starts in a system with a Red Giant. If we can possibly kick up the possibility of failure to 11 that’d be awesome possum. 

Preliminary scans of [planet] indicate a canyon with several openings throughout. We originally thought they may be dwellings for some or another type of wildlife, but we now are not so sure as one of [scientist’s] sub-drones realised that they are spaced almost equidistantly apart from one another. Not only that but each one—though poorly constructed—is a square too similar to the others to be considered a natural construction. 

Two probes sent down show a heavy door rolled over a second entrance about 100 meters into each cave, very well preserved. Unknown glyphs and runes line the walls leading to these entrances completely floor-to-ceiling.

[Send a ground unit and start standard excavation procedures]

Stage 1 (Difficulty=2)

We began with four different archways located in the most accessible points within the canyon. The entrance is heavy, but not immobile. Standard archaeological procedure was able to roll the first one aside. Though the others are proving a bit more stubborn, it’s believed that more force will eventually yield the intended result. Of note, we did find clues of triggers for several traps that are more likely than not still functional. Extreme caution advised; some of these look quite nasty indeed. 

For now, we can focus our attention on the first room. It’s a tomb we’ve dated pretty far back to the end of this culture’s stone age. It looks to be completely untouched except for dust and contains a multitude of small personal items and clay jars filled with the remnants of food. First speculation of course is that these are the occupant’s intended supplies to help them into the afterlife. Translation work of the text on the rough stone sarcophagus in the center of the room have given us their initial work, and it reads: “Forever rests Xulathun X, Shaman of the Red and the highest of highs.” 

However, text in a different language was found on the archway above and behind the tomb in clear view of the door. This was remarkably easier to translate and just simply read: “You can go home”. It has been suggested that earlier grave robbers trying to pull a joke, but if that’s the case, why didn’t they take anything? 

[Maintain work on the other entrances]

Failure: While trying to work the stone, one of the drones fell into one a rather classical trap. They tripped a pressure plate and were immediately rewarded with an impalement that shocked the whole team to the core. As their position was untenable for recovery, they were put down. 

[Inconvenient. Continue.] 

Reward: +1 minor relic. 

Stage 2 (Difficulty=6)

Almost immediately after we managed to gain entry to the first tomb, the second doorway started to give way under our techniques and then was easily rolled aside. However, this then gave way to another door down a short hallway. This next door is cast in mostly red marble with an inscription above in a language totally divorced from that of the first tomb, the script more rigid and angled. Even a cursory examination made it apparent that we’re looking at two completely different culture sets. 

Work began on opening this door and proceeding inside. The locks were robust, but not terribly difficult to figure out with some improvised manipulation. Multiple drones reported hearing voices, though none of them appear to have heard the same set of phrases. None of these voices have been recorded on any piece of equipment, though multiple team members have tried. Multiple traps were identified, most of them triggered by tripwire. 

The electric lights that powered on upon entering the tomb were surprising as were the remarkably clean air readings. The sarcophagus itself in this case was encased in glass all the way around allowing a clear view of the mummified corpse within, a mammalian creature similar in height to us, with neatly-groomed facial hair and wearing a rotted and—for such a revered leader—simple garment. The translators have finally worked out the script that is over the entrance to this tomb, which reads simply “Lenin” and also in a plaque on the wall in here reading “Revolutioneer. Hero of the Supreme Soviet. Lover of the Red”. Above the tomb itself was text in the same style as the cryptic message in Xulathun X’s tomb reading simply “you can still go home”. 

Failure 1: Feeling around while examining the door, a researcher drone reported feeling a directional tuft of air, followed by the sounds of struggle and weapons fire from what sound files correspond to roughly industrial age weaponry. The entire group was found slaughtered upon further investigation with a single fully-loaded firearm placed in between their bodies as if placed as a statement. The ammunition canister was fully loaded and in a curved shape. No signs of the assailants were found. 

[Curious.] (-5 clues, +1 relic)

Failure 2: The paleontology drone at base camp, while performing tests on what was thought to be a fossil, exploded into a fiery rage screeching about the rights of the workers and the upheaval of the monied classes. He then picked up a hatchet and proceeded to hack his fellow drones to death. He was finally subdued by the guards who were apparently malfunctioning for an unknown reason as they were not at their posts within the camp. The entire away team is troubled to say the least. The object the consultant was examining turned out to be a small fossilized piece of jewelry, most likely a lapel; a small, red shiny pentagram.

[Destroy the malfunctioning drones.] (-7 clues, +1 relic)

Reward: +3 minor relics, +200 society research

Stage 3 (Difficulty=7)

We were caught somewhat in shock as we managed to slide the stone away from the next door. She was greeted with a huge wall of water! Past that however, we were clearly able to see inside to a room that seems to be somewhat larger than the last two. After a few hours of planning it was agreed that we should proceed forward with diving equipment, an unexpected but quickly fulfilled request. 

After donning our bathing costumes, we entered the tomb and it is clear that this one is much more artistically-focused than the others. A large stone sarcophagus lies against the northern wall, before which are two smaller stone sarcophagi and twelve still smaller ones in  neat rows alternating directions so if lines were drawn they are in a perpendicular grid. It would appear that each one was treated to grow algae and coral, and there are waterproofed sunlamps adoring the roof to help maintain the growth. There are beautiful intricate pattern spawning twisted branches of colour and material. Many colors are present but predictably the dominant one here is red and its various shades. Still immaculately beautiful. 

As with the previous tombs there is an inscription on the wall, these inlaid with crimson bio luminescent microorganisms seemingly to make the words glow. It reads: “Sacrificed here for the Red is the Lifetime Archduke of the Oceans Glublureum of Saltsand and his wives and children. Given unto the red for peace and fulfillment, loved by his supplicants, defeater of the prawn beasts”. And on the opposite wall in the same script as the other warnings. “Next comes the bargain. You can still go home.”

At this point, a large sinkhole has opened up in the middle of the dig site taking a small bulldozer that was parked there near it. Looking down into it, we can see a circular stone floor with what is most likely a handle in the middle of it. Spelunking would easily drop us down onto the surface. 

[Make it so]

Failure: After getting the door to the tomb open, one or our crew leaned up against a wall and managed to trigger some kind of switch which dumped onto everybody’s heads a multitude of relatively benign looking but actually very vicious crabs. The pincers were quite poisonous. After drawing back the team, we have a total death count of 4 with all 9 others injured. The bodies are severely bloated and hemorrhaging and it’s debatable how many of the drones will make it through the night let alone the next week. 

[Unfortunate] (-5 clues)

Reward: +500 society research

**Comment by Z8MB1E** [comment]
I've already begun coding this event chain, but I have a question: when you say "possibility of failure", which of these (if any) do you mean:

1. Possibility of failure for anomaly to even spawn,
2. Possibility of failure for investigating scientist to spawn the digsite after researching anomaly,
3. Possibility of failure for each stage of the digsite to result in a negative outcome?

If none of these, what do you mean in particular?

**Comment by whitewolf87** [comment]
The third one. The idea is that it'll have a very high likelihood of spawning every failure event during the course of the dig site. I'm piggybacking on the failure mechanic to enhance the story. Kind of supposed to give a real horror vibe and reinforce the notion that the ultimate outcome is due to player choice. Like they could indeed pack up at any time and F the F off home.

**Comment by Z8MB1E** [comment]
> 
> 
> The third one. The idea is that it'll have a very high likelihood of spawning every failure event during the course of the dig site. I'm piggybacking on the failure mechanic to enhance the story. Kind of supposed to give a real horror vibe and reinforce the notion that the ultimate outcome is due to player choice. Like they could indeed pack up at any time and F the F off home.

Sounds good — I'll put it together under that pretense. :)

**Comment by Z8MB1E** [comment]
I've created a new `echoes_of_the_red` branch with the relevant files committed. Feel free to checkout that branch and test it in your own game — I've been testing in mine and it works pretty well! A few things to note:

- The banner cannot change colors without the entire flag changing (including symbol). I didn't want to make an empire do that as it might upset someone who created a custom empire only for their entire color scheme to irreversibly shift because of some dig site they uncovered, as well as the loss of a preferred symbol on the flag.
- Localization was updated to make a bit more sense in some areas as well as additional creative writing and some grammatical fixes. For the most part, everything is what you've written, and the actual story line has not changed whatsoever.

If you have any additional questions, feel free to let me know!
To spawn the anomaly manually, just find and select a planet, then use `create_anomaly mem_red_echoes_category` in the console. If you want to match the standard starting conditions, find a habitable planet orbiting a red giant, then trigger the anomaly creation. From that point on, it's all yours!

**Comment by whitewolf87** [comment]
Frikkin' sweet!

I'm a bit disappointed by the lack of color change ability, but you are right; if I'm honest I'd be pissed if I worked super hard on my flag and it changed on me. It would have been an interesting visual effect, but I don't think there's too much lost without it. 

Your other changes I'm sure are fine, indeed thanks bunches for everything including the grammar fixes. 

I've got another writing project (non-Stellaris-related) that I need to start cooking, and also some business out of town, but I'll be able to test the event myself in a couple of days time, and I'm sure it'll be dandy. I might actually be able to do it on my laptop tonight or tomorrow.
